index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="UTF-8">
     <title>12 o'clocks</title>
+    <link rel="shortcut icon" href="http://tdctokyo.org/eng/wp/wp-content/uploads/2010/08/1acc8a9dfa88b5ab93da8d26fd93a95c.jpg"/>
```

Comment:
I'm not sure about this icon image - there are usually fairly strict requirements on favicons (Particularly that they are quite low resolution. Also, it'd be better if it was stored locally on the github.

Reply:
I agree, I mainly just did this as a example